src/config/index.tsx

```diff
@@ -1,9 +1,7 @@
 import Constants from "expo-constants";
 
 export const SENTRY_DSN: string = Constants.manifest?.extra?.sentryDsn;
-export const IS_STORYBOOK_VIEW =
-  Constants.manifest?.extra?.storybook ||
-  (Constants.manifest?.releaseChannel || "").indexOf("storybook") > -1;
```

Comment:
https://u.expo.dev/cf511b6b-f66b-45b5-b856-2339182f8052?runtime-version=exposdk:45.0.0&channel-name=storybook-pr660&platform=android release channel isnt found in the manifest anymore